ee/modules/610-service-with-healthchecks/docs/README_RU.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Модуль service-with-healthchecks"
+description: "Readiness-пробы с модулем service-with-healthchecks в Deckhouse Kubernetes Platform"
+---
+
+В Kubernetes трафик направляется только на доступные поды, что достигается с помощью ресурсов типа `Service` и `EndpointSlices` — это обеспечивает высокую доступность и надежность приложения, так как только  готовые к обработке запросов поды принимают трафик.
```

Comment:
In the first sentence there must be declaration, that active pods are selected with Service and readiness probes. And under the hood the `EndpointSlices` are being generated.